### DISCUSSION TITLE
Audio from Mopidy/Snapclient is not playing on my bluetooth speaker

### DISCUSSION THREAD

**Discussion by malatu14** [snapcast/snapcast#1001] [Q&A]
Hi,

I'm trying to use an old HP TX1000 computer as a Snapclient, using an Debian distribution. Bluetooth is correctly enabled on it but for some reason, songs streamed from my snapserver (an RPi) is not broacasted on my bluetooth speaker, but on the internal speaker of the laptop. Otherway, all sounds (watching videos from Youtube or sounds) are correctly played on my bluetooth speaker.

Do you have an idea to solve it ?

Thanks you !

**Comment by tabearunzheimer** [comment]
Did you get this running? What's your OS?

My first suggestion would be to make sure to set your Bluetooth speaker as the default device.
